backend/internal/controller/adapter/shot_message.go

```diff
@@ -52,11 +52,17 @@ func UnpackShotMsg(message io.Reader) (model.Shot, error) {
 	}
 
 	return model.Shot{
-		Speed: params.Speed,
-		Team:  teamID,
+		Speed:      params.Speed,
+		Team:       teamID,
+		ShotAtGoal: checkIfShotAtGoal(params.EndArea, teamID),
```

Comment:
You have changed returned structure in UnpackShotMsg() but haven't changed tests for the function.

Reply:
Tests have been modified

Reply:
Good job sir